rag_ingestion/ingest_bank_docs.py

```
@@ -0,0 +1,103 @@
+                loan_product_id, 
+                full_doc_name, 
+                i, # chunk_index
+                content, 
+                embedding_vector
+            ))
+            
+            if (i + 1) % 5 == 0:
+                print(f"  -> {i + 1}개 적재 진행 중...")
+
+        conn.commit()
+        print(f"\n✅ 성공: 모든 청크가 'nudgebank' DB에 적재되었습니다!")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)